cli/src/klio_cli/cli.py

```diff
@@ -139,9 +139,6 @@ def run_job(klio_config, config_meta, **kwargs):
         config_meta.job_dir, kwargs.get("image_tag")
     )
     image_tag = kwargs.get("image_tag") or git_sha
-    if config_meta.config_file:
-        basename = os.path.basename(config_meta.config_file)
-        image_tag = "{}-{}".format(image_tag, basename)
```

Comment:
The reason this was in here was to be able to tell which config the image was built with (for those using multiple configuration files). I'm not sure I understand the reasoning behind removing it?

Reply:
hmm that's an interesting point.  I did this because I thought the reason it was included in the tag name was to make it easier to know which config file was included in the image as the `effectiive-klio-job.yaml`.  Since that's no longer used, there's no longer a need for that kind of indication.

I do agree though that it would be good to know what config was used when the image was built, however now that I think about it, adding the file name to the image tag would still result in some ambiguity because:
1.  You don't know what version of the file was used, only the name
2.  Now that we support overrides, it could be misleading if the user provides a CLI arg to `klio image build` that affects the build process

So I still kinda think this change makes sense, but maybe it would be a good idea to include a `build-config.yaml` or something in the image?

Reply:
Those are fair arguments. I think a `build-config.yaml` makes sense too. Perhaps a separate ticket, so we can get this PR through?

Reply:
yeah I agree that can be addressed separately